[2024-08-07T22:22:08.327+0000] {processor.py:161} INFO - Started process (PID=31102) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:22:08.328+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:22:08.329+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:22:08.329+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:22:08.341+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:22:08.339+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:22:08.342+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:22:08.354+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.030 seconds
[2024-08-07T22:22:38.582+0000] {processor.py:161} INFO - Started process (PID=31161) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:22:38.583+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:22:38.585+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:22:38.584+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:22:38.594+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:22:38.592+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:22:38.595+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:22:38.609+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.031 seconds
[2024-08-07T22:23:09.051+0000] {processor.py:161} INFO - Started process (PID=31220) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:23:09.052+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:23:09.053+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:23:09.053+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:23:09.061+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:23:09.060+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:23:09.062+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:23:09.075+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.028 seconds
[2024-08-07T22:23:39.493+0000] {processor.py:161} INFO - Started process (PID=31279) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:23:39.493+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:23:39.494+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:23:39.494+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:23:39.503+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:23:39.501+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:23:39.503+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:23:39.516+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.027 seconds
[2024-08-07T22:24:09.652+0000] {processor.py:161} INFO - Started process (PID=31338) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:24:09.653+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:24:09.654+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:24:09.654+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:24:09.662+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:24:09.660+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:24:09.663+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:24:09.675+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.027 seconds
[2024-08-07T22:24:40.015+0000] {processor.py:161} INFO - Started process (PID=31397) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:24:40.016+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:24:40.017+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:24:40.017+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:24:40.024+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:24:40.022+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:24:40.025+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:24:40.036+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.025 seconds
[2024-08-07T22:24:53.665+0000] {processor.py:161} INFO - Started process (PID=31442) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:24:53.665+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:24:53.667+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:24:53.667+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:24:53.680+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:24:53.678+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:24:53.680+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:24:53.693+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.032 seconds
[2024-08-07T22:25:00.154+0000] {processor.py:161} INFO - Started process (PID=31457) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:25:00.154+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:25:00.155+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:25:00.155+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:25:00.169+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:25:00.167+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:25:00.170+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:25:00.185+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.035 seconds
[2024-08-07T22:25:30.459+0000] {processor.py:161} INFO - Started process (PID=31516) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:25:30.459+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:25:30.461+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:25:30.460+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:25:30.469+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:25:30.467+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:25:30.470+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:25:30.483+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.028 seconds
[2024-08-07T22:26:00.708+0000] {processor.py:161} INFO - Started process (PID=31575) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:26:00.709+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:26:00.710+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:26:00.710+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:26:00.719+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:26:00.717+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:26:00.720+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:26:00.732+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.027 seconds
[2024-08-07T22:26:30.922+0000] {processor.py:161} INFO - Started process (PID=31634) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:26:30.923+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:26:30.924+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:26:30.924+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:26:30.932+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:26:30.930+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:26:30.933+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:26:30.946+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.028 seconds
[2024-08-07T22:27:01.095+0000] {processor.py:161} INFO - Started process (PID=31693) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:27:01.096+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:27:01.097+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:27:01.097+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:27:01.105+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:27:01.103+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:27:01.105+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:27:01.119+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.028 seconds
[2024-08-07T22:27:31.253+0000] {processor.py:161} INFO - Started process (PID=31752) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:27:31.254+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:27:31.255+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:27:31.255+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:27:31.263+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:27:31.261+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:27:31.264+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:27:31.278+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.029 seconds
[2024-08-07T22:28:01.488+0000] {processor.py:161} INFO - Started process (PID=31811) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:01.489+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:28:01.490+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:28:01.490+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:01.498+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:28:01.496+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:28:01.498+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:01.511+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.027 seconds
[2024-08-07T22:28:03.515+0000] {processor.py:161} INFO - Started process (PID=31812) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:03.515+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:28:03.517+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:28:03.516+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:03.529+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:28:03.527+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:28:03.530+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:03.542+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.031 seconds
[2024-08-07T22:28:14.652+0000] {processor.py:161} INFO - Started process (PID=31819) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:14.652+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:28:14.654+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:28:14.653+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:14.666+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:28:14.664+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:28:14.667+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:14.680+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.032 seconds
[2024-08-07T22:28:20.299+0000] {processor.py:161} INFO - Started process (PID=31836) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:20.300+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:28:20.301+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:28:20.301+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:20.314+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:28:20.312+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:28:20.315+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:20.327+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.032 seconds
[2024-08-07T22:28:47.002+0000] {processor.py:161} INFO - Started process (PID=31881) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:47.002+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:28:47.004+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:28:47.003+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:47.017+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:28:47.015+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:28:47.017+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:28:47.030+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.032 seconds
[2024-08-07T22:29:04.907+0000] {processor.py:161} INFO - Started process (PID=31932) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:29:04.907+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:29:04.909+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:29:04.908+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:29:04.921+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:29:04.919+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:29:04.922+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:29:04.934+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.031 seconds
[2024-08-07T22:29:29.631+0000] {processor.py:161} INFO - Started process (PID=31982) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:29:29.632+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:29:29.634+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:29:29.633+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:29:29.647+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:29:29.645+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:29:29.647+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:29:29.660+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.033 seconds
[2024-08-07T22:29:59.750+0000] {processor.py:161} INFO - Started process (PID=32041) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:29:59.751+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:29:59.752+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:29:59.752+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:29:59.760+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:29:59.758+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:29:59.761+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:29:59.774+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.027 seconds
[2024-08-07T22:30:29.891+0000] {processor.py:161} INFO - Started process (PID=32100) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:30:29.892+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:30:29.893+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:30:29.893+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:30:29.900+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:30:29.899+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:30:29.901+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:30:29.914+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.026 seconds
[2024-08-07T22:30:42.456+0000] {processor.py:161} INFO - Started process (PID=32116) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:30:42.457+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:30:42.458+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:30:42.458+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:30:42.472+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:30:42.470+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:30:42.473+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:30:42.487+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.035 seconds
[2024-08-07T22:31:12.595+0000] {processor.py:161} INFO - Started process (PID=32175) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:31:12.596+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:31:12.597+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:31:12.597+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:31:12.606+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:31:12.604+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:31:12.606+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:31:12.619+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.028 seconds
[2024-08-07T22:31:34.423+0000] {processor.py:161} INFO - Started process (PID=32225) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:31:34.424+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:31:34.425+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:31:34.425+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:31:34.436+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:31:34.434+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:31:34.436+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:31:34.449+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.030 seconds
[2024-08-07T22:32:04.580+0000] {processor.py:161} INFO - Started process (PID=32284) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:32:04.580+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:32:04.582+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:32:04.581+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:32:04.589+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:32:04.588+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:32:04.590+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:32:04.602+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.027 seconds
[2024-08-07T22:32:05.602+0000] {processor.py:161} INFO - Started process (PID=32285) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:32:05.602+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:32:05.603+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:32:05.603+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:32:05.616+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:32:05.615+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:32:05.617+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:32:05.630+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.033 seconds
[2024-08-07T22:32:35.811+0000] {processor.py:161} INFO - Started process (PID=32344) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:32:35.811+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:32:35.813+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:32:35.812+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:32:35.822+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:32:35.820+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:32:35.823+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:32:35.837+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.030 seconds
[2024-08-07T22:33:05.916+0000] {processor.py:161} INFO - Started process (PID=32403) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:33:05.916+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:33:05.918+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:33:05.917+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:33:05.926+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:33:05.924+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:33:05.926+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:33:05.939+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.028 seconds
[2024-08-07T22:33:36.154+0000] {processor.py:161} INFO - Started process (PID=32462) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:33:36.155+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:33:36.156+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:33:36.156+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:33:36.164+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:33:36.162+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:33:36.164+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:33:36.176+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.025 seconds
[2024-08-07T22:34:06.684+0000] {processor.py:161} INFO - Started process (PID=32521) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:34:06.684+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:34:06.686+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:34:06.686+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:34:06.700+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:34:06.698+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:34:06.701+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:34:06.714+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.035 seconds
[2024-08-07T22:34:37.368+0000] {processor.py:161} INFO - Started process (PID=32580) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:34:37.369+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:34:37.370+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:34:37.370+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:34:37.378+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:34:37.376+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:34:37.378+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:34:37.391+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.027 seconds
[2024-08-07T22:35:07.947+0000] {processor.py:161} INFO - Started process (PID=32639) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:35:07.947+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:35:07.948+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:35:07.948+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:35:07.955+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:35:07.954+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:35:07.956+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:35:07.968+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.025 seconds
[2024-08-07T22:35:38.391+0000] {processor.py:161} INFO - Started process (PID=32698) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:35:38.392+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:35:38.395+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:35:38.394+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:35:38.404+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:35:38.402+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:35:38.405+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:35:38.418+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.032 seconds
[2024-08-07T22:36:08.994+0000] {processor.py:161} INFO - Started process (PID=32757) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:36:08.995+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:36:08.996+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:36:08.996+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:36:09.003+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:36:09.002+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:36:09.004+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:36:09.017+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.027 seconds
[2024-08-07T22:36:39.721+0000] {processor.py:161} INFO - Started process (PID=32816) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:36:39.721+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:36:39.723+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:36:39.723+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:36:39.730+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:36:39.729+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:36:39.731+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:36:39.744+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.028 seconds
[2024-08-07T22:37:09.993+0000] {processor.py:161} INFO - Started process (PID=32875) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:37:09.994+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:37:09.995+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:37:09.995+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:37:10.002+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:37:10.001+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:37:10.003+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:37:10.015+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.026 seconds
[2024-08-07T22:37:40.073+0000] {processor.py:161} INFO - Started process (PID=32934) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:37:40.073+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:37:40.074+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:37:40.074+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:37:40.082+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:37:40.080+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:37:40.083+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:37:40.095+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.026 seconds
[2024-08-07T22:38:10.265+0000] {processor.py:161} INFO - Started process (PID=32993) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:38:10.266+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:38:10.267+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:38:10.267+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:38:10.275+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:38:10.273+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:38:10.275+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:38:10.288+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.027 seconds
[2024-08-07T22:38:40.388+0000] {processor.py:161} INFO - Started process (PID=33052) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:38:40.389+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:38:40.390+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:38:40.390+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:38:40.397+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:38:40.395+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:38:40.397+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:38:40.411+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.027 seconds
[2024-08-07T22:39:10.507+0000] {processor.py:161} INFO - Started process (PID=33111) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:39:10.507+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:39:10.509+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:39:10.508+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:39:10.516+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:39:10.515+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:39:10.517+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:39:10.536+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.033 seconds
[2024-08-07T22:39:40.605+0000] {processor.py:161} INFO - Started process (PID=33170) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:39:40.606+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:39:40.607+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:39:40.607+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:39:40.615+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:39:40.613+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:39:40.616+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:39:40.629+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.027 seconds
[2024-08-07T22:40:10.786+0000] {processor.py:161} INFO - Started process (PID=33229) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:40:10.787+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:40:10.788+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:40:10.788+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:40:10.795+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:40:10.794+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:40:10.796+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:40:10.808+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.025 seconds
[2024-08-07T22:40:40.948+0000] {processor.py:161} INFO - Started process (PID=33288) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:40:40.949+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:40:40.950+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:40:40.950+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:40:40.958+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:40:40.956+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:40:40.958+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:40:40.971+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.026 seconds
[2024-08-07T22:41:11.043+0000] {processor.py:161} INFO - Started process (PID=33347) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:41:11.044+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:41:11.045+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:41:11.045+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:41:11.053+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:41:11.051+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:41:11.053+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:41:11.065+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.025 seconds
[2024-08-07T22:41:41.239+0000] {processor.py:161} INFO - Started process (PID=33406) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:41:41.240+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:41:41.241+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:41:41.241+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:41:41.248+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:41:41.247+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:41:41.249+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:41:41.260+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.026 seconds
[2024-08-07T22:42:11.386+0000] {processor.py:161} INFO - Started process (PID=33465) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:42:11.387+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:42:11.388+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:42:11.388+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:42:11.395+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:42:11.394+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:42:11.396+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:42:11.409+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.028 seconds
[2024-08-07T22:42:41.516+0000] {processor.py:161} INFO - Started process (PID=33524) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:42:41.516+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:42:41.518+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:42:41.518+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:42:41.538+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:42:41.535+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:42:41.539+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:42:41.557+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.046 seconds
[2024-08-07T22:43:11.635+0000] {processor.py:161} INFO - Started process (PID=33583) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:43:11.636+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:43:11.637+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:43:11.637+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:43:11.645+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:43:11.643+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:43:11.646+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:43:11.659+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.028 seconds
[2024-08-07T22:43:41.975+0000] {processor.py:161} INFO - Started process (PID=33642) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:43:41.975+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:43:41.977+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:43:41.976+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:43:41.984+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:43:41.982+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:43:41.984+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:43:41.996+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.026 seconds
[2024-08-07T22:44:12.103+0000] {processor.py:161} INFO - Started process (PID=33701) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:44:12.104+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:44:12.105+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:44:12.105+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:44:12.112+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:44:12.111+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:44:12.113+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:44:12.126+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.027 seconds
[2024-08-07T22:44:42.220+0000] {processor.py:161} INFO - Started process (PID=33760) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:44:42.220+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:44:42.221+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:44:42.221+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:44:42.229+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:44:42.227+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:44:42.230+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:44:42.242+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.026 seconds
[2024-08-07T22:45:12.562+0000] {processor.py:161} INFO - Started process (PID=33825) to work on /opt/airflow/dags/dag_script.py
[2024-08-07T22:45:12.563+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/dag_script.py for tasks to queue
[2024-08-07T22:45:12.564+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:45:12.564+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/dag_script.py
[2024-08-07T22:45:12.571+0000] {logging_mixin.py:188} INFO - [2024-08-07T22:45:12.569+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/dag_script.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_script.py", line 4, in <module>
    from clickhouse_driver import Client
ModuleNotFoundError: No module named 'clickhouse_driver'
[2024-08-07T22:45:12.572+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_script.py
[2024-08-07T22:45:12.583+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/dag_script.py took 0.025 seconds
